### ISSUE TITLE
Two Intermission activities launched from CountdownTimer

### ISSUE THREAD

**Issue by kriskrunch** [kriskrunch/OneMoreTime#4]
Major problem with multiple activities being launched from the intermission.  Worse, is that there is an old Intermission activity that continues to appear, even though I've changed all of the names, "Stephanie" is still appearing as the "next player up".  Perhaps changing everything from using shared preferences to using the bundle exclusively is the answer.

**Comment by kriskrunch** [comment]
First, I tried preventing multiple activities following the suggestions listed here: https://stackoverflow.com/questions/10614565/how-to-avoid-multiple-instances-of-same-activity.  

That didn't work.  It just causes the app to crash now. Let's try to move everything into the bundle instead of using the shared preferences.

If that doesn't work, we may have to create a single activity that oscillates between two layouts in code.

Is the button being pressed twice? I don't think so because I made the onClick function disable the button after the first press...
